codes/go/chapter_hashing/array_hash_map.go

```diff
@@ -0,0 +1,96 @@
+// File: array_hash_map.go
+// Created Time: 2022-12-14
+// Author: msk397 (machangxinq@gmail.com)
+
+package chapter_hashing
+
+import "fmt"
+
+/* 键值对 int->String */
+type Entry struct {
+	key int
+	val string
+}
+
+/* 基于数组简易实现的哈希表 */
+type ArrayHashMap struct {
+	bucket []*Entry
+}
+
+func newArrayHashMap() *ArrayHashMap {
+	// 初始化一个长度为 100 的桶（数组）
+	bucket := make([]*Entry, 100)
+	return &ArrayHashMap{bucket: bucket}
+}
+
+/* 哈希函数 */
+func (a *ArrayHashMap) hashFunc(key int) int {
+	index := key % 100
+	return index
+}
+
+/* 查询操作 */
+func (a *ArrayHashMap) get(key int) string {
+	index := a.hashFunc(key)
+	pair := a.bucket[index]
+	if pair == nil {
+		return "Not Found"
```

Comment:
Could we return nil here?

Please carefully check the correspondence to the reference Language.